app/controllers/api/account.php

```diff
@@ -602,7 +644,7 @@
                         'email' => $email,
                         'emailVerification' => true,
                         'status' => true, // Email should already be authenticated by OAuth2 provider
-                        'password' => Auth::passwordHash(Auth::passwordGenerator(), Auth::DEFAULT_ALGO, Auth::DEFAULT_ALGO_OPTIONS),
+                        'password' => null,
```

Comment:
Can we ensure those accounts without password can do password-protected actions? Like change password, or delete project

Reply:
Yes, things work better if password is null.